src/core/weightedGraph.test.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import * as Weights from "./weights";
+import {Graph} from "./graph";
+import * as WeightedGraph from "./weightedGraph";
+import * as GraphTest from "./graphTestUtil";
+
+describe("core/weightedGraph", () => {
+  function expectEqual(wg1, wg2) {
+    expect(wg1.graph.equals(wg2.graph)).toBe(true);
+    expect(wg1.weights).toEqual(wg2.weights);
+  }
+  it("empty produces an empty WeightedGraph", () => {
```

Comment:
Nit: to prevent noise when adding tests, should use `describe()` consistently.
```
describe "empty"
  it "produces ____"
```

Reply:
I think it's fine, since we don't have any other tests for empty, nor is it clear why we would add them. If/when we do add another test case, indenting this one test is not that noisy.